mlir/utils/jenkins/Jenkinsfile

```diff
@@ -878,6 +879,11 @@ pipeline {
                                     }
                                 }
                             }
+                            //Accuracy_checker will compare outputs from MIGraphX and onnx runtime
+                            dir('MIGraphX/tools/accuracy') {
+                                sh 'pip3 install -r requirements.txt'
```

Comment:
I'm not happy with this CI-time install.

Could you move the requirements into the relevant Dockerfile instead?

Reply:
Thank you, I will remove that from Jenkinsfile and add to Dockerfile.